pkg/operatormanager/operatormanager.go

```diff
@@ -496,9 +502,10 @@ func (m *OperatorManager) createOrUpdateSidecarsConfig(ctx context.Context, name
 }
 
 func (m *OperatorManager) createOrUpdateSidecarsConfigMap(ctx context.Context, namespace string, c *v1.ConfigMap, objs []client.Object) ([]client.Object, error) {
+	sidecarsCMName := c.Data["per-namespace-sidecars-configmap-name"]
 
 	sccm := &v1.ConfigMap{}
-	if err := m.SetName(sccm, pg.SidecarsCMName); err != nil {
```

Comment:
Maybe move this `var` to operatormanager.

Reply:
As above, when *go* code and *yaml* share a variable, use the one in the *yaml*.

Reply:
As above, the name is dynamically derived.

Reply:
now it's getting even more complicated? we create that configmap ourselves, and then derive it's name from another config we also created?

Reply:
Final decision: Use hard-coded `const`.